src/lib/make-toolbox-xml.js

```diff
@@ -485,18 +485,18 @@ const sensing = `
         <block type="sensing_mousedown"/>
         <block type="sensing_mousex"/>
         <block type="sensing_mousey"/>
-        <block type="sensing_loudness"/>
-        <block type="sensing_timer"/>
+        <block id="loudness" type="sensing_loudness"/>
+        <block id="timer" type="sensing_timer"/>
         <block type="sensing_resettimer"/>
-        <block type="sensing_of">
+        <block id="of" type="sensing_of">
```

Comment:
Is this one supposed to be monitorable?

Reply:
Seems to be, it is a change from scratch 2